detailed_docs/SDL-HMI_API/GetSystemTime/GetSystemTime_TRS.md

```diff
@@ -18,14 +18,16 @@ and HMI sends valid response during `<DefaultTimeout>`
 SDL must  
 
 - perform DTLS Handshake with mobile app  
+- send OnServiceUpdate(service_Type, REQUEST_ACCEPTED) notification to HMI  
 - send START_SERVICE_ACK, encryption = true to mobile app  
  
 3. In case  
 SDL gets GetSystemTime response from HMI  
 and mobile and SDL certificates are not valid according to date/time from GetSystemTime response  
 
 SDL must  
-- trigger PTU   
+- trigger PTU  
+- send OnServiceUpdate(service_Type, INVALID_CERT)   
```

Comment:
@KhrystynaDubovyk This message will be sent along with `START_SERVICE_NACK` and `in case of receiving one more time expired certificate during PTU`

Reply:
@dboltovskyi, I'm not quite sure if I got you right... 
please, check  https://github.com/smartdevicelink/sdl_requirements/pull/123/commits/b8f6f100b744b2a00b53e02c5a903af93067601f

Reply:
@KhrystynaDubovyk Let me put some clarification here
Currently we have the following statements:
> - send OnServiceUpdate(service_Type, INVALID_CERT) notification to HMI    
> - respond START_SERVICE_NACK, encryption = false to mobile app in case of receiving one more time expired certificate during PTU

I understand these in a following way:
1) SDL certainly sends OnServiceUpdate(service_Type, INVALID_CERT) without any conditions
2) SDL sends START_SERVICE_NACK only in case of receiving one more time expired certificate during PTU

However SDL wil send both OnServiceUpdate(service_Type, INVALID_CERT) and START_SERVICE_NACK only in case of receiving one more time expired certificate during PTU

Reply:
@dboltovskyi, thanks for the clarification. please, have a look https://github.com/smartdevicelink/sdl_requirements/pull/123/commits/f8332eb7bd7edfbda78439c56304e7a0465641db